tests/foreman/cli/test_computeresource.py

```diff
@@ -395,18 +428,20 @@ def test_set_console_password_v2(self, set_console_password):
         Targets BZ 1100344.
 
         """
-        name = gen_string('utf8')
-        result = ComputeResource.create({
-            'name': name,
-            'provider': 'Libvirt',
-            'url': gen_url(),
-        })
-        self.assertEqual(result.return_code, 0)
-        self.assertEqual(len(result.stderr), 0)
-
-        result = ComputeResource.update({
-            'name': name,
-            'set-console-password': set_console_password,
-        })
-        self.assertEqual(result.return_code, 0)
-        self.assertEqual(len(result.stderr), 0)
+        for set_console_password in ('true', 'false'):
+            with self.subTest(set_console_password):
+                name = gen_string('utf8')
+                result = ComputeResource.create({
```

Comment:
Move the object creation outside the loop and perform all updates inside the loop to speed up the test.